saul-examples/src/main/java/edu/illinois/cs/cogcomp/saulexamples/data/CLEFImageReader.java

```diff
@@ -251,51 +253,56 @@ private void getSegmentsRelations(String spatial_rels) throws IOException
                                 val = (int) topo[x][y];
                                 if (val == 1)
                                     rel = "adjacent";
-                                else if (val == 2)
-                                    rel = "disjoint";
+//                              ****** Ignoring as not required in mSpRL
+//                                else if (val == 2)
+//                                    rel = "disjoint";
```

Comment:
The whole point of the data model is that we should be able to make such filterings using the data model, not changing the reader.